src/style.css

```diff
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search--form {
+  margin: 20px 0;
+}
+
+.searchbar {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  display: flex;
+  width: 100%;
+
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #404e5f;
+}
+
+.big {
+  height: 70px;
+  font-size: 16px;
+  padding: 25px 62px 23px;
```

Comment:
Add only left padding and use line-height for set the text on center (it should fix tests)